easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,23 @@
+name = 'SWIG'
+version = '3.0.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d']
+
+gccver = '6.4.0'
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('PCRE', '8.41','',('GCCcore', gccver)),
```

Comment:
@molden Please don't hardcode to a particular subtoolchain, just use `('PCRE', '8.41')`